### PR TITLE
docs(dashboard): document dashboard and dashboard cards

### LINKED ISSUE #96: [Story] Document each type of card

### PR DESCRIPTION
**Pull request by maxcao13**
Fixes #96 #95

### CODE REVIEW

File: guides/_subsections/navigate-the-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+        text="Click the expandable menu on <i>New Layout</i> button and select <i>Choose Template</i>. This will open the template picker."
+        %}
+    </li>
+    <li>
+        {% capture template-picker-guide-text %}
+        <p>
+            The template picker displays all the available templates. Templates are categorized into 3 groups.
+            <ol>
+                <li>
+                    <b>Suggested:</b> Templates that are suggested for you based on recent activity.
```
Comment (andrewazores) [inline guides/_subsections/navigate-the-dashboard.md:383]: Is this actually based on recent activity? I missed that part...

File: guides/_subsections/navigate-the-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+        caption="Click and drag the <i>Target JVM Details</i> card's header on top or to the right of the <i>MBeans Metrics Chart</i> card to swap their positions."
+        %}
+    </li>
+    <li>
+        {% capture remove-card-text %}
+            <p>
+                Each card header contains a kebab icon that opens a menu of card actions. The actions menu contains the following options:
+                <ul>
+                    <li>
+                        <b>View</b> - Opens the card in widescreen mode.
```
Comment (andrewazores) [inline guides/_subsections/navigate-the-dashboard.md:289]: Fullscreen maybe, rather than widescreen?

**Review by andrewazores — COMMENTED** [review]
Content looks awesome, nice job.

File: guides/_subsections/navigate-the-dashboard.md (hunk 1 above)
Comment (maxcao13) [reply guides/_subsections/navigate-the-dashboard.md:383]: Well technically it is based on your most recently used layouts, we just keep the last 5 recently used templates in localStorage, but it sounds cooler if we say "recent activity" and its technically somwhat true? No ML component at all towards this :-)

**Pull request merged**